examples/slash_commands/main.go

```diff
@@ -0,0 +1,120 @@
+package main
+
+import (
+	"flag"
+	dgo "github.com/bwmarrin/discordgo"
```

Comment:
`go fmt` ? U need split up sdk import and local. It's a good code style

Reply:
I tried gofmt, it seems I need to run it again or do it manually.

Reply:
Need update